### PR TITLE
Treasury update

### PR DESCRIPTION
**Pull request by lwjohnst86**

### CODE REVIEW

**Comment by lwjohnst86** [comment]
@linamnt you mind taken a quick look over and approving?

**Comment by linamnt** [comment]
sorry was I supposed to keep it open :S? @lwjohnst86

File: treasurer/README.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Values surrounded by brackets `()` denote negative values, as is often done in a
...
 
-| Item      |       Amount|
-|:----------|------------:|
-| Snacks    |    ($270.62)|
-| Misc      |       ($200)|
-| Workshops |    $1,455.76|
-| **Total** |  **$985.14**|
+| Item      |         Amount|
+|:----------|--------------:|
+| Snacks    |      ($193.03)|
```
Comment (QuLogic) [inline treasurer/README.md]: Why did snacks go down? Was this an estimate?

Comment (QuLogic) [reply treasurer/README.md]: Oh, of course, after posting I see it's in the projections...

**Pull request merged**